cmd/nfd-topology-updater/main.go

```diff
@@ -133,7 +133,7 @@ func initFlags(flagset *flag.FlagSet) (*topology.Args, *resourcemonitor.Args) {
 	flagset.StringVar(&args.KubeConfigFile, "kubeconfig", "",
 		"Kube config file.")
 	flagset.DurationVar(&resourcemonitorArgs.SleepInterval, "sleep-interval", time.Duration(60)*time.Second,
-		"Time to sleep between CR updates. Non-positive value implies no CR updatation (i.e. infinite sleep). [Default: 60s]")
+		"Time to sleep between CR updates. zero means no CR updates on interval basis. [Default: 60s]")
```

Comment:
We might want to update the description in docs (docs/reference/topology-updater-commandline-reference.md) too(?)

Also nit: the commit msg for this is a bit misleading. Adjust a bit e.g. simply `topology-updater: disable sleep-interval` -> `topology-updater: make it possible to disable sleep-interval`(?)

Reply:
Done.